src/mixins/renderer.js

```diff
@@ -37,10 +37,6 @@ export default function (incrementalDom) {
             this.renderToken(tokens, i, options)()
           }
         })
-        if (!preventEndHTMLParser) {
-          iDOMParser.end()
-          iDOMParser.reset()
-        }
```

Comment:
Each of `renderToIncrementalDOM` and `renderInlineToIncrementalDOM` calls mixin method `render` and it always prevents to close parse object in `renderInline`.

It means user never reach to these logic.